apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -48,15 +48,22 @@ internal sealed class TargetedSessionPool : SessionPoolBase
             // Tasks waiting for the pool to reach its minimum size or become unhealthy.
             private readonly LinkedList<TaskCompletionSource<int>> _minimumSizeWaiters = new LinkedList<TaskCompletionSource<int>>();
 
+            // The size of the last unhealthy batch.
+            // If 0, the pool is healthy.
             // Assume the pool is healthy to start with.
-            private int _healthy = 1;
-            private bool Healthy => Interlocked.CompareExchange(ref _healthy, 0, 0) == 1;
+            private int _lastUnhealthyBatch;
```

Comment:
Possibly `_lastUnhealthyBatchSize` to make it clear that it's the size? (Ditto the property.)